docs/getting-started/configure-via-openshift-ui.md

```diff
@@ -0,0 +1,59 @@
+## Configuring VDO via Openshift Web Console
+
+The easiest way to configure VDO is to configure the CRD's via Openshift Web Console, follow the steps below for the configuration.
```

Comment:
The easiest way to configure VDO is through VDOCTL. we highly recommend downloading the VDO command line utility to configure VDO. However you can still configure VDO through the Openshit UI. Please follow the below steps to configure VDO from Openshift UI